Atomic/delete.py

```diff
@@ -77,6 +94,11 @@ def delete_image(self):
         for del_obj in delete_objects:
             del_obj.backend.delete_image(del_obj.input_name, force=self.args.force)
 
+        if not self.args.force:
+            # for the used images, output the error messages
+            for used_img in used_images:
+                util.write_out("Image {} is being used by a container currently".format(used_img.id[0:12]))
```

Comment:
Maybe "Image {} is currently being used by container {}" and display the container and image name?